### PR TITLE
Fix Last Will Crash(es)

### PR DESCRIPTION
**Pull request by Spandamn**
Any opinions on what the error message can be changed to?

### CODE REVIEW

File: config/formats.js [hunk 1]
```diff
@@ -297,8 +297,12 @@ exports.Formats = [
 		ruleset: ['[Gen 7] OU'],
 		banlist: ['Endeavor', 'Blast Burn + Explosion + Frenzy Plant + Giga Impact + Hydro Cannon + Hyper Beam + Self Destruct + V-Create > 2'],
 		onBeforeFaint: function (pokemon, source) {
-			this.add('-hint', `${pokemon.name || pokemon.species}'s Last Will let it use one last move!`);
-			this.runMove(pokemon.moves[pokemon.moves.length - 1], pokemon);
+			try {
+				this.add('-hint', `${pokemon.name || pokemon.species}'s Last Will let it use one last move!`);
+				this.runMove(pokemon.moves[pokemon.moves.length - 1], pokemon);
+			} catch (e) {
+				this.add('-hint', `Oops! ${pokemon.name || pokemon.species}'s Last Will caused a crash. You can still continue playing the battle, though.`);
```
Comment (Zarel) [inline config/formats.js]: You don't need to be _that_ specific.

Maybe just "But it failed to complete!" or something.

**Comment by Zarel** [comment]
Damn it, TI, that's not the message I wanted either.

**Comment by Zarel** [comment]
Specifically, it doesn't work because Fury Cutter will do damage, and then crash, so "failed" is not entirely appropriate here.

**Comment by Zarel** [comment]
(Because it did everything you expected it to do)

**Comment by TheImmortal** [comment]
Then why not have no message at all

**Comment by Zarel** [comment]
It's unclear if any move will fail entirely...

**Pull request merged**